docs/guis/betabeat/model_creation.md

```diff
@@ -0,0 +1,260 @@
+# Model Creation
+
+## Model Selection Window
+
+The _model selection window_ **opens automatically** after starting the main GUI frame from the [beam selection window](beam_selection.md).
+This window can be opened at any time by clicking the **left-most button at the top** of the GUI, depicting a "model".
+
+<figure>
+  <center>
+  <img src="../../assets/images/betabeat_gui/top_buttons.png" width="100%" alt="Icons at the top of the GUI." />
+  <figcaption>Open the model creation window with the left-most button at the top, depicting a model.</figcaption>
+  </center>
+</figure>
+
+
+If there have been no models previously created, the window will look empty like this:
+
+<figure>
+  <center>
+  <img src="../../assets/images/betabeat_gui/model_creation_window.png" width="100%" alt="Model selection window." />
+  <figcaption>Blank Model selection window.</figcaption>
+  </center>
+</figure>
+
+
+After [creating a model](#model-creation-window), the window will show the **folder names** in the _Models_ folder of the
+accelerator sub-folder selected in the [beam selection window](beam_selection.md).
+
+<figure>
+  <center>
+  <img src="../../assets/images/betabeat_gui/model_creation_with_model.png" width="100%" alt="Model selection window with created model." />
+  <figcaption>Model selection window with a created model</figcaption>
+  </center>
+</figure>
```

Comment:
Could maybe be a tabbed view to avoid jumping down then up quite a bit (yes I click all the blue words). Very minor